package-res/resources/web/pentaho/util/arg.js

```diff
@@ -20,23 +20,73 @@ define([
 
   var A_slice = Array.prototype.slice;
 
+  /**
+   * @name pentaho.util.arg
+   * @class
+   * @amd pentaho/util/arg
+   *
+   * @classDesc Util methods for keyword args.
+   *
+   * The AMD module returns the arg utils.
+   *
+   * @description Performs actions on keyword args.
+   */
   return {
+    /**
+     * Provides the optional property value of a given object or the given default value if the given object does
+     * not contain the sought after property value.
+     *
+     * @param {?object} [obj] The object to search for given propName param.
+     * @param {String} [propName] Property name to find in obj param.
+     * @param {any} [defaultValue] The default value to return if propName param is not found. The default value of this param is undefined and is considered specified when its value is not a nully value. {@link Nully}
+     *
```

Comment:
Please do not extend the text of argument descriptions much, like what you did with the `defaultValue`.
It's generally preferable to have a paragraph in the main description, describing the behavior of the function, even if the paragraph focus on a single argument.
The description of arguments, should focus more on immediate and obvious stuff...
Something like this:
```js
/**
 * @param {?object} obj The object from which to get a property.
 * @param {string} propName The name of the property.
 * @param {any} [defaultValue] The default value.
```